src/actions/main.js

```diff
@@ -1,82 +0,0 @@
-export const CHECK_LOCATION = "CHECK_LOCATION"
```

Comment:
Not in all projects. I'm currently using it because it's the new trending stack , but I have angularJS projects in es3 and plain vanilla js projects in es3 too.